src/App/Fossa/ListTargets.hs

```diff
@@ -97,14 +100,49 @@ runAll ::
   , Has (Reader AllFilters) sig m
   , Has Telemetry sig m
   ) =>
+  ListTargetOutputFormat ->
   Path Abs Dir ->
   m ()
-runAll basedir = traverse_ single discoverFuncs
+runAll outputFmt basedir = traverse_ single discoverFuncs
   where
-    single (DiscoverFunc f) = withDiscoveredProjects f basedir (printSingle basedir)
+    single (DiscoverFunc f) = withDiscoveredProjects f basedir formatter
```

Comment:
[note] We can use the `Output (Discovered Project a)` to sink across thread boundaries as mentioned in ticket's implementation note, but I chose to `logStdout`'s concurrent output for simplicity - but happy to change it!

Ref: https://hackage.haskell.org/package/concurrent-output-1.10.19/docs/System-Console-Concurrent.html#v:outputConcurrent

Reply:
This seems OK to me as long as the data is output atomically and not mangled.